
---
string[]  srv_JointName
string[]  srv_JointName_hiTask
float32[] srv_JointPos_hiTask
float32[] srv_ExeTime_hiTask